packages/mui-joy/src/SvgIcon/SvgIconProps.ts

```diff
@@ -0,0 +1,82 @@
+import { OverridableStringUnion, OverrideProps } from '@mui/types';
+import * as React from 'react';
+import { SxProps } from '../styles/defaultTheme';
+import { SvgIconClasses } from './svgIconClasses';
+
+export interface SvgIconPropsSizeOverrides {}
+
+export interface SvgIconPropsColorOverrides {}
+
+export interface SvgIconTypeMap<P = {}, D extends React.ElementType = 'svg'> {
+  props: P & {
+    /**
+     * Node passed into the SVG element.
+     */
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the component.
+     */
+    classes?: Partial<SvgIconClasses>;
+    /**
+     * The color of the component. It supports those theme colors that make sense for this component.
+     * You can use the `htmlColor` prop to apply a color attribute to the SVG element.
+     * @default 'inherit'
+     */
+    color?: OverridableStringUnion<
+      'inherit' | 'primary' | 'neutral' | 'warning' | 'success' | 'danger' | 'info',
+      SvgIconPropsSizeOverrides
+    >;
+    /**
+     * The fontSize applied to the icon. Defaults to 24px, but can be configure to inherit font size.
+     * @default 'medium'
+     */
+    fontSize?: OverridableStringUnion<
+      'inherit' | 'extraSmall' | 'small' | 'medium' | 'large' | 'extraLarge',
```

Comment:
@hbjORbj I was using `small`, ...`large` before but I agree with marija that using T-shirt (`sm`, `md`, ...) size is more consistent for Joy.

```suggestion
      'inherit' | keyof FontSize,
```

you can import the `FontSize` from `styles/types/typography`. This is a better way because if developer augment the FontSize in the theme, the icon will be augmented as well.

Reply:
I agree !